integration/netmon/netmon_test.bats

```diff
@@ -0,0 +1,54 @@
+#!/usr/bin/env bats
+#
+# Copyright (c) 2018 Intel Corporation
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+load "${BATS_TEST_DIRNAME}/../../lib/common.bash"
+
+# Environment variables
+IMAGE="busybox"
+NETWORK_NAME="test"
+CONTAINER_NAME="containerA"
+PAYLOAD_ARGS="tail -f /dev/null"
+
+setup() {
+	clean_env
+
+	# Check that processes are not running
+	run check_processes
+	echo "$output"
+	[ "$status" -eq 0 ]
+
+	extract_kata_env
+
+	# Enabling netmon at the configuration file
+	sed -i 's|#enable_netmon = true|enable_netmon = true|' ${RUNTIME_CONFIG_PATH}
+}
+
+@test "test netmon" {
+	docker network create $NETWORK_NAME
```

Comment:
Nit: either you choose to put `docker network create` in `setup()` to match with `docker network rm` in `teardown()`, or you can have both `docker network create` and `docker network rm` in the main test function.